BootloaderCommonPkg/Include/Library/MultibootLib.h

```diff
@@ -234,11 +234,16 @@ typedef struct {
   BOOTPARAMS_IMAGE_DATA   BootParamsData;
 } TRUSTY_IMAGE_DATA;
 
+typedef struct {
+  struct multiboot2_start_tag         *start_tag;
+} MULTIBOOT2_INFO;
```

Comment:
This library header file is an interface for other modules to use Multiboot library.
so this library should not depend on library implementation including structure definitions.
For other modules (e.g. OsLoader), it looks "struct multiboot2_start_tag" is not defined in API file.
and prefer updating the variable name "start_tag" in this file to follow SBL variable style (e.g. StartTag).

Reply:
"struct multiboot2_start_tag" is opaque from a client point of view (for using the library it's only necessary that it's a pointer to a structure but not the structure definition).  I'll change the header file to stress that.

Reply:
Thanks for the update.